DataProcessor/src/net/relinc/processor/controllers/StrainGaugeController.java

```diff
@@ -286,6 +290,7 @@ else if(folderCount > 0)
 		}
 		else{
 			if(Dialogs.showConfirmationDialog("Deleting Strain Gauge", "Confirm", "Are you sure you want to delete this strain gauge?", stage))
+				new File(file.getPath() + ".json").delete();
 				new File(file.getPath() + ".txt").delete();
```

Comment:
I though this might throw an exception since one won't exist. I tested and it does not.